plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/advanced_query_on_log_set/action.py

```diff
@@ -173,7 +176,18 @@ def maybe_get_log_entries(
 
         results_object = response.json()
         if statistical:
-            potential_results = results_object.get("partial")
+            stats_endpoint = f"{self.connection.url}log_search/query/{results_object.get('id', '')}"
+            self.logger.info(f"Getting statistical from: {stats_endpoint}")
+            stats_response = self.connection.session.get(stats_endpoint, params=params)
+            try:
+                stats_response.raise_for_status()
+            except Exception:
```

Comment:
Exception is very broad here. Maybe narrow it down.
```suggestion
            except HTTPError as error:
```